src/Forms/SearchableDropdownTrait.php

```diff
@@ -319,7 +319,7 @@ public function getAttributes(): array
      */
     public function getValueArray(): array
     {
-        $value = $this->Value();
+        $value = $this->getValue();
```

Comment:
In `ListboxField::getValueArray()` you used `getFormattedValue()` - why do we use the raw value here? Is one of these incorrect, or are they both right?

If they're both right, probably add a in-code comment to each explaining why that is the method used, so we don't notice the discrepency in the future and think something's wrong.

Reply:
Changed ListboxField::getValueArray() to use getValue()